custom_components/hacs/api/hacs_repository_data.py

```diff
@@ -41,6 +41,9 @@ async def hacs_repository_data(hass, connection, msg):
         if repo_id in hacs.common.skip:
             hacs.common.skip.remove(repo_id)
 
+        # Strip trailing slash character
+        repo_id = repo_id.rstrip("/")
```

Comment:
This will work for that specific example only.
Instead of this, it should be regex to ensure that we only get the parts we need